host/dkms.conf

```diff
@@ -0,0 +1,8 @@
+PACKAGE_NAME=xap-hss-host
```

Comment:
How about just `hsst`, not `xap-hss-host`?

Reply:
Changed